src/algo_dicho.jl

```diff
@@ -0,0 +1,108 @@
+# MIT License
+# Copyright (c) 2017: Xavier Gandibleux, Anthony Przybylski, Gauthier Soleilhac, and contributors.
+
+function solve_dicho(m::JuMP.Model; round_results::Bool=false, verbose, kwargs...)
```

Comment:
`round_results` is a keyword argument here instead of an argument, same for verbose.